test/js/ui/map.test.js

```diff
@@ -56,6 +56,11 @@ test('Map', (t) => {
         t.ok(map.keyboard.isEnabled());
         t.ok(map.scrollZoom.isEnabled());
         t.ok(map.touchZoomRotate.isEnabled());
+        t.throws(() => {
+            new Map({
+                container: 'anElementIdWhichDoesNotExistInTheDocument'
+            });
+        }, Error, 'throws on invalid map container id');
```

Comment:
> Although the test I added just checks that an Error is thrown not the exact message, but an Error was already thrown before, so the test I added actually passes without this change.

You can change `Error` to a regex or string that matches the desired error message. This will prevent other errors (i.e. `Map is not defined`) from causing this test to pass when an undesired error is thrown. 

See the tap assert docs for `t.throws` and `t.match` http://www.node-tap.org/asserts/